From c84f4206d3634983330ec658f9a384cdece5b25c Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 6 Nov 2024 11:59:37 +0100
Subject: [PATCH 754/841] fix(fdts): set flexgen 58 to HSE_KER for
 STM32MP257F-EV1

Flexgen 58 must set with HSE_KER for low power management

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e1ff03ebc583113b54c95243c74a5b4373921e65 fix(fdts): set flexgen 58 to HSE_KER for STM32MP257F-EV1]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I383c753b5af19a193c7911f591985830724bb0b6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/416771
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index 77fb413f3..1f6ded5e0 100644
--- a/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -50,7 +50,7 @@
 		FLEXGEN_CFG(48, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(51, XBAR_SRC_PLL4, 0, 5)
 		FLEXGEN_CFG(52, XBAR_SRC_PLL4, 0, 5)
-		FLEXGEN_CFG(58, XBAR_SRC_HSE, 0, 1)
+		FLEXGEN_CFG(58, XBAR_SRC_HSE_KER, 0, 1)
 		FLEXGEN_CFG(63, XBAR_SRC_PLL4, 0, 2)
 	>;
 
-- 
2.47.2

